app/src/main/java/com/hci3/aris/ui/balance/BalanceFragment.java

```diff
@@ -27,9 +31,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         return inflater.inflate(R.layout.fragment_balance, container, false);
     }
 
+
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-    }
 
+        String[] titles = new String[]{"Balance", "Enrollment", "Scholarship"};
+        TabLayout tabLayout;
+        ViewPager2 viewPager2;
+
+        tabLayout = getView().findViewById(R.id.tabBar);
+        viewPager2 = getView().findViewById(R.id.viewPager);
+
+        BalanceAdapter adapter = new BalanceAdapter(this);
```

Comment:
Since 3 na ung parameter nung `BalanceAdapter` na pinabago ko kanina, magiging:

```java
BalanceAdapter balanceAdapter = new BalanceAdapter(getContext(), this, tabLayout.getTabCount());
```